development/tools/cmd/pjtester/README.md

```diff
@@ -0,0 +1,49 @@
+# Prowjob tester
+
+Prowjob tester is a tool for testing changes to Prowjobs and scripts in the `test-infra` repository which are under development. It uses the production Prow instance to run chosen Prowjobs with code from pull requests (PRs) without going through multiple cycles of new PRs, reviews, and merges. The whole development is done within one PR.
```

Comment:
It's spelled `ProwJob`. Please correct it in the whole document. I corrected it only in the bits that I suggested changes to for other reasons.

It would be a good idea to consider that in the name of the tool, too, unless it's already got a fixed spelling. 

```suggestion
Prowjob tester is a tool for testing changes to ProwJobs and scripts in the `test-infra` repository which are under development. It uses the production Prow instance to run chosen ProwJobs with code from pull requests (PRs) without going through multiple cycles of new PRs, reviews, and merges. The whole development is done within one PR.
```